helm/app/values-prod.yaml

```diff
@@ -1,2 +1,3 @@
-replicaCount: 0 # prod should be offline until approval to go live is given
-
+secureRoute:
+  wwwRoute:
+    enable: true
```

Comment:
this will certainly work, and we may find that as a rule we should always be adding the `www.` subdomain to all routes regardless of if the route represents apex or sub